sdk/servicebus/tests.yml

```diff
@@ -12,13 +12,11 @@ jobs:
     parameters:
       ServiceDirectory: servicebus
       EnvVars:
-        SERVICE_BUS_HOSTNAME: $(python-sb-livetest-service-bus-hostname)
-        SERVICE_BUS_SAS_POLICY: $(python-sb-livetest-service-sas-policy)
-        SERVICE_BUS_SAS_KEY: $(python-sb-livetest-service-sas-key)
-        SERVICE_BUS_CONNECTION_STR: $(python-sb-livetest-service-connection-str)
-        SERVICE_BUS_CONNECTION_STR_RO: $(python-sb-livetest-service-connection-str-ro)
-        SERVICE_BUS_CONNECTION_STR_WO: $(python-sb-livetest-service-connection-str-wo)
-        SERVICE_BUS_CONNECTION_STR_ENTITY: $(python-sb-livetest-service-connection-entity)
+        AZURE_SUBSCRIPTION_ID: $(azure-subscription-id)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_CLIENT_OID: $(aad-azure-sdk-test-client-oid)
```

Comment:
Looking at our mgmt_settings_real, we appear to consume Subscription, Tenant, Client id, and Client Secret, but not OID.

Thanks for the help reconciling this @danieljurek! (don't hesitate to ping if any other things I can help with.)